gfx/layers/client/TiledContentClient.cpp

```diff
@@ -146,7 +146,10 @@ SharedFrameMetricsHelper::UpdateFromCompositorFrameMetrics(
 {
   MOZ_ASSERT(aLayer);
 
-  CompositorChild* compositor = CompositorChild::Get();
+  CompositorChild* compositor = nullptr;
+  if (XRE_GetProcessType() != GeckoProcessType_Default) {
```

Comment:
I remember we had a problem with process types in embedlite that don't match the process types in B2G. Hopefully this code path isn't touched in embedlite.
Otherwise LGTM.